documentation/source/user_section/tutorials/other/spinalcord-smoothing.rst

```diff
@@ -0,0 +1,34 @@
+.. TODO:
+
+   Is this one-page tutorial necessary? It is basically just telling users that the ``sct_smooth_spinalcord`` tool exists. (Compared to other tutorials, which demonstrate multi-step workflows.)
+
+   So, I am thinking that maybe this page will be unnecessary once we organize the "Command-Line Tools" page into one-page-per-script. We could simply have all of this information on the dedicated "sct_smooth_spinalcord" page instead, and save the "Tutorials" for complex workflows only.
```

Comment:
I'm a little suspicious of the 2 one-page tutorials. Instead, this material could be included in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3095.